test/test_transaction.py

```diff
@@ -114,56 +135,50 @@ def test_execute(self, mock_time):
         request.slave_id = 1
         request.function_code = 222
         trans = SyncModbusTransactionManager(client, 0.3, False, False, 3)
-        trans._recv = mock.MagicMock(  # pylint: disable=protected-access
-            return_value=b"abcdef"
-        )
+        setattr(trans, "_recv", mock.MagicMock(return_value=b"abcdef"))
```

Comment:
This is just tricking `mypy` and `pylint` which AFAIK don't process `setattr()`.

Cute, but not much advantage over `# type: ignore`.